core-api/src/main/java/com/sequenceiq/cloudbreak/validation/Permission.java

```diff
@@ -58,11 +61,35 @@
 
     private String description;
 
-    Permissions(Resource resource, Action action, String description) {
+    Permission(Resource resource, Action action, String description) {
```

Comment:
Should be renamed to OrganizationPermission(s) since it's tied to the organization.
There should be a separate class for the tenant permissions.